README.md

```diff
@@ -157,17 +160,16 @@ spec:
           claimName: pravega-tier2
 ```
 
-where:
```

Comment:
This is not a note. It's a description of the variables in the example snippet. So I should leave it as it is.